Files/DataStandard/Vault.Custom/addinVault/powerGateMain.ps1

```diff
@@ -1,9 +1,6 @@
 ﻿$global:addinPath = $PSScriptRoot
 Import-Module "C:\ProgramData\coolOrange\powerGate\Modules\Initialize.psm1" -Global
 Initialize-CoolOrange
-
-ConnectToErpServerWithMessageBox
```

Comment:
Why has this code been removed in Vault ? is the integration still working without this code block in Vault tabs?

Reply:
@TollJulian see my question.

Reply:
I removed these codelines because they leaded into four additional calls Connect-ERPWithMessageBox.

Reply:
also the ERP tabs work as usual